### ISSUE TITLE
Cancel button of CtkInputDialog is bugged

### ISSUE THREAD

**Issue by cursecatcher** [TomSchimansky/CustomTkinter#1280]
Hello everyone, 

I found a small bug in CtkInputDialog class. 
Both the **ok** and the **cancel** buttons are associated with the **ok event**. 
As a consequence, the value inserted in the CtkEntry is returned regardless of which button is actually pressed. 
You can reproduce the bug by running the following code:

```
import customtkinter as ctk 

input_dialog = ctk.CTkInputDialog( 
    title="just a test", 
    text="insert a value" )
value = input_dialog.get_input()
print(f"Inserted value is {value}")
```

**Comment by AronLenvig** [comment]
I found the bug:
 self._cancel_button = CTkButton(master=self,
                                        width=100,
                                        border_width=0,
                                        fg_color=self._button_fg_color,
                                        hover_color=self._button_hover_color,
                                        text_color=self._button_text_color,
                                        text='Cancel',
                                        command=self._ok_event)
        self._cancel_button.grid(row=2, column=1, columnspan=1, padx=(10, 20), pady=(0, 20), sticky="ew")
        
 command=self.ok_event should be command=self._cancel_event)

**Comment by Wolf-SO** [comment]
> I found the bug [...]

This change is already known and addressed by the pending pull request #1281

**Comment by TomSchimansky** [comment]
fixed with version 5.2
